app/test_main.py

```diff
@@ -1 +1,14 @@
 # write your code here
+from app.main import can_access_google_page
+from unittest import mock
+
+
+def test_can_access_google_page():
+    with (
+        mock.patch("current_time") as mocked_current_time,
+        mock.patch("response") as mocked_response
```

Comment:
You are mocking 'current_time' and 'response' but in the wrong way. You should mock them where they are defined. For example, if 'current_time' is a function in the 'app.main' module then it should be mocked as 'app.main.current_time'. Same applies to 'response'.